youtube_dl/extractor/bbc.py

```diff
@@ -761,8 +762,13 @@ class BBCIE(BBCCoUkIE):
         'only_matching': True,
     }, {
         # custom redirection to www.bbc.com
+        # video with window.__INITIAL_DATA__
         'url': 'http://www.bbc.co.uk/news/science-environment-33661876',
-        'only_matching': True,
+        'id': 'p02xzws1',
+        'title': 'Pluto may have \'nitrogen glaciers\'',
+        'description': 'Pluto could have glaciers of nitrogen ice, new photographs from Nasa\'s New Horizons probe suggest.',
+        'thumbnail': 'https://c.files.bbci.co.uk/29DE/production/_84481701_84481700.jpg',
```

Comment:
Do not use exact URLs in tests.

Reply:
How about a regex there, say `r're:https?://.+/.+\.jpg'`?

Thx for review, I'll push an update and post you when it's up.

Reply:
>How about a regex there, say `r're:https?://.+/.+\.jpg'`?

This is fine.

Reply:
Seems to be a recurrent CI problem where `pip install nose` fails in the Jython VMs? If that's going to work next time, I'd like to push a rebased update.

Reply:
Jython issue is intermittent you can ignore it. Also there is no need to rebase as GitHub does this just fine.